ci/generate-test-media.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+set -e
+
+cd build
+
+# Create core video, audio, and subtitle streams to construct files out of.
+ffmpeg -y -f lavfi -i testsrc=duration=2:size=1280x720 video.mkv
+ffmpeg -y -f lavfi -i sine=frequency=1000:duration=2 audio.flac
```

Comment:
Wouldn't be better to do it with mpv?

Reply:
I would personally prefer avoiding mpv's encoding mode.